### PR TITLE
Add functionality to auto-inject the authorization token

### PR DESCRIPTION
**Pull request by pmuens**
See: https://github.com/serverless/product/issues/319

### CODE REVIEW

File: lib/utils/headers.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const keys = require('ramda/src/keys')
+const contains = require('ramda/src/contains')
+const assoc = require('ramda/src/assoc')
+
+module.exports = {
+  injectAuthorizationToken: headers => {
+    let updatedHeaders = headers
```
Comment (elsteelbrain) [inline lib/utils/headers.js]: I think this should be `Object.assign({}, headers)` if you want `headers` to be not affected by changes.

File: lib/utils/headers.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const keys = require('ramda/src/keys')
+const contains = require('ramda/src/contains')
+const assoc = require('ramda/src/assoc')
+
+module.exports = {
+  injectServerlessApplicationToken: headers => {
+    let updatedHeaders = headers
```
Comment (elsteelbrain) [inline lib/utils/headers.js]: I think this should be `Object.assign({}, headers)` if you want `headers` to be not affected by changes.

Comment (pmuens) [reply lib/utils/headers.js]: Good point! :+1:

Or I do smth like:

```javascript
updatedHeaders = {
  ...headers
}
```

Comment (elsteelbrain) [reply lib/utils/headers.js]: Yeah, that would work too.

Comment (pmuens) [reply lib/utils/headers.js]: Hehe. You're solution is better here since the FDK doesn't sport object destructuring. Will update the PR in a few seconds.

**Comment by pmuens** [comment]
@elsteelbrain thanks for the quick review!

I updated the code to address your concerns regarding immutability :+1:

**Comment by pmuens** [comment]
Merging this...

**Pull request merged**